schemas/openapi.yml

```diff
@@ -394,6 +455,7 @@ components:
           description: Input prompt for the step.
           type: string
           example: Write the words you receive to the file 'output.txt'.
+          nullable: true
```

Comment:
didn't want to add an RFC just for that: people don't necessarilly want to put an input here.